.travis.yml

```diff
@@ -103,41 +47,12 @@ matrix:
       script:
         - make
         - AWS_KVS_LOG_LEVEL=7 ./kvspic_test --gtest_break_on_failure --gtest_filter="-TimerQueueFunctionalityTest.*:HeapPerfTest.*:ThreadFunctionalityTest.ThreadCreateAndCancel"
-
-    # Old Version GCC 4.4
-    - name: "Linux GCC 4.4 Build"
-      os: linux
-      addons:
-        apt:
-          sources:
-            - sourceline: 'ppa:ubuntu-toolchain-r/test'
-          packages:
-            - gcc-4.4
-            - gdb
-      compiler: gcc
-      before_script: export CC=gcc-4.4 && mkdir build && cd build && cmake .. -DBUILD_TEST=TRUE
-
-    # Cross-compilation to ARM, no tests are run
-    - name: "ARM Cross-compilation"
-      os: linux
-      addons:
-        apt:
-          packages:
-            - gcc-arm-linux-gnueabi
-            - g++-arm-linux-gnueabi
-            - binutils-arm-linux-gnueabi
-      compiler: gcc
-      before_script:
-        - export CC=arm-linux-gnueabi-gcc CXX=arm-linux-gnueabi-g++
-        - mkdir build && cd build
-        - cmake .. -DBUILD_TEST=TRUE
-      script: make
-
+        
     # Windows build
-    - name: "Windows MSVC"
-      os: windows
-      script:
-          - unset CC CC_FOR_BUILD CXX CXX_FOR_BUILD # We want to use MSVC
-          - .github/build_windows.bat
-          - export "AWS_KVS_LOG_LEVEL=7"
-          - cd build && ./kvspic_test.exe --gtest_filter="-TimerQueueFunctionalityTest.*:HeapPerfTest.*:PermutatedStreamInfo/StateTransitionFunctionalityTest.ControlPlaneServiceCallExhaustRetry*:PermutatedStreamInfo/IntermittentProducerAutomaticStreamingTest.ValidateTimerInvokedBeforeTime*:PermutatedStreamInfo/IntermittentProducerAutomaticStreamingTest.ValidateTimerInvokedAfterFirstPeriod*:PermutatedStreamInfo/IntermittentProducerAutomaticStreamingTest.ValidateLastUpdateTimeOfStreamUpdated*:PermutatedStreamInfo/IntermittentProducerAutomaticStreamingTest.MultiTrackVerifyNoInvocationsWithSingleTrackProducer*:PermutatedStreamInfo/IntermittentProducerAutomaticStreamingTest.ValidateNoConsecutiveEOFR*:PermutatedStreamInfo/IntermittentProducerAutomaticStreamingTest.ValidateErrorOnForceConsecutiveEOFR*"
+    # - name: "Windows MSVC"
```

Comment:
Remove or uncomment based on Windows working in GHA.

Reply:
Removed in the latest commit